mock/mockmr/views.go

```diff
@@ -209,7 +209,7 @@ func (e *Engine) Execute(opts ExecuteOptions) (int, *ExecuteResults, error) {
 	}
 
 	var output []outputItem
-	if opts.Reduce {
+	if opts.Reduce && view.ReduceFunc != "" {
```

Comment:
what happens if someone sets reduce=true but there is no reduce func? (hopefully it returns the same error the server does? :D

Reply:
I actually thought this wasn't an error case - turns out it is.